xtas/core.py

```diff
@@ -84,7 +84,7 @@ def configure(config, import_error="raise", unknown_key="raise"):
 
     app.config_from_object(config.get('CELERY', 'xtas._defaultconfig.CELERY'))
     es = config.get('ELASTICSEARCH', _defaultconfig.ELASTICSEARCH)
-    _config['ELASTICSEARCH'] = es
+    config['ELASTICSEARCH'] = es
```

Comment:
Ah, I missed that, pretty silly considering it's right there on line 86. So maybe rename the parameter new_config?